restaurants/src/app/components/restaurantdetails/restaurantdetails.component.html

```diff
@@ -0,0 +1,13 @@
+<div *ngIf="restaurant">
+    <h2>{{restaurant.name | uppercase}} Details</h2>
+    <div>
+      <label for="restaurant-name">Restaurant name: </label> 
+        {{restaurant.name}}
+    </div>
```

Comment:
Just in terms of good (or more standard) html formatting - I wouldn't put text straight in a div that also has another child element already (in this case the <div> starting on line 3 contains the child element <label> and then the free text on line 5). Separating out elements will make styling more clear/easy when you get to doing that. Instead, I'd recommend

```
<div>
      <label for="restaurant-name">Restaurant name: </label> 
      <div id="restaurant-name">{{restaurant.name}}<div>
</div>
```

or

```
<div>
      <label for="restaurant-name">Restaurant name: </label> 
      <span id="restaurant-name">{{restaurant.name}}<span>
</div>
```

Probably lean towards using the latter, but doesn't really matter.  I also added an id on that element - generally recommend doing that for values or inputs that are displayed, it's a good habit to get into - it makes writing automated tests a lot easier down the line too.